lib/src/mt_dev.c

```diff
@@ -1871,6 +1871,21 @@ struct mt_tx_queue* mt_dev_get_tx_queue(struct mtl_main_impl* impl, enum mtl_por
 
   mt_pthread_mutex_lock(&inf->tx_queues_mutex);
   for (uint16_t q = 0; q < inf->max_tx_queues; q++) {
+    if (ST21_TX_PACING_WAY_TSN == inf->tx_pacing_way) {
```

Comment:
igc driver enabled launch time TX at queue 0 for TSN-based pacing.
So, any traffic requiring launch time TX feature is allocated queue 0.

Reply:
Sounds like it's a WA to igc only, can you limit it to MT_DRV_IGC? 
```
if ((ST21_TX_PACING_WAY_TSN == inf->tx_pacing_way) && (MT_DRV_IGC == inf->drv_info.drv_type)) 
```

This is necessary as we will has other NICs to support TSN pacing also.

Reply:
Done.